pkg/server/problem_ranges.go

```diff
@@ -289,7 +289,7 @@ const debugProblemRangesTemplate = `
         <DIV CLASS="row">
           {{- if $.LeaderNotLeaseholder}}
             {{- range $_, $r := $.LeaderNotLeaseholder}}
-              <a href="/debug/range?id={{$r}}">{{$r}}</a>
+              <a href="/debug/range?id={{$r}}">{{$r}}</a> &nbsp;
```

Comment:
I want to keep the `-` in `{{- end}}` or else it adds an extra newline (not visible, but make the source ugly).  But you're right, I can remove the breaking space.  Done.

Notice the space difference with the previous screencap.

<img width="647" alt="screen shot 2017-04-28 at 15 03 13" src="https://cloud.githubusercontent.com/assets/1614265/25543287/d632fd20-2c23-11e7-9f8d-c38061b65475.png">